environment

```diff
@@ -55,6 +55,8 @@ DSS_GS_CHECKOUT_BUCKET_TEST=org-humancellatlas-dss-checkout-test
 DSS_GS_CHECKOUT_BUCKET_TEST_FIXTURES=org-humancellatlas-dss-checkout-test-fixtures
 DSS_GS_CHECKOUT_BUCKET_STAGING=org-humancellatlas-dss-checkout-staging
 DSS_GS_CHECKOUT_BUCKET_PROD=org-humancellatlas-dss-checkout-prod
+DSS_GS_EVENT_RELAY_AWS_ACCESS_KEY_ID=
+DSS_GS_EVENT_RELAY_AWS_SECRET_ACCESS_KEY=
```

Comment:
I don't think you should clobber these variables here. Did you set environment variables in the Travis UI? If so, their values won't be available since you clobber them here. You should probably leave a comment here instead, naming these variables and prompting the operator to set them when deploying.

Reply:
Yes, the variables were set in the Travis UI. Will un-clobber and note.